### PR TITLE
Add redaction field to invocation table

### PR DESCRIPTION
**Pull request by bduffany**
---

**Version bump**: Patch

### CODE REVIEW

File: server/tables/tables.go [hunk 1]
```diff
@@ -98,10 +98,14 @@ type Invocation struct {
...
-	UploadThroughputBytesPerSecond   int64
-	ActionCount                      int64
-	Perms                            int   `gorm:"index:perms"`
+	DurationUsec                   int64
+	UploadThroughputBytesPerSecond int64
+	ActionCount                    int64
+	Perms                          int `gorm:"index:perms"`
+	// The level of redaction applied to this invocation when it was written,
+	// where each bit corresponds to the type of content being redacted (ex:
+	// BuildBuddy API key)
```
Comment (tylerwilliams) [inline server/tables/tables.go:107]: rm

**Review by tylerwilliams — APPROVED** [review]
LGTM

(Please apply this to dev db before committing)

File: server/tables/tables.go (hunk 1 above)
Comment (bduffany) [reply server/tables/tables.go]: Done

**Comment by bduffany** [comment]
Applied to dev DB

**Pull request merged**